packages/components/src/SquiggleChart.tsx

```diff
@@ -38,11 +42,13 @@ export const SquiggleChart : React.FC<SquiggleChartProps> = props => {
   }
  
 
-  let result = run(props.squiggleString, samplingInputs);
-  console.log(result)
+  let result = run(props.squiggleString, samplingInputs, props.environment);
   if (result.tag === "Ok") {
-    let chartResults = result.value.map(chartResult => {
-      console.log(chartResult)
+    let environment = result.value.environment
+    let exports = result.value.exports
+    if(props.onEnvChange)
```

Comment:
There should be a space between "if" and the "(" sign. I recommend using a formatter tool; there's one build into the Rescript plugin of VSCode.

Reply:
What? This is typescript not Rescript...

Reply:
Sorry; I still think you could format it though.

Reply:
we'll have the _impose lint as a build step_ conversation tomorrow, hopefully I'll learn how to write a `package.json` fast enough to impose it quickly.

Reply:
(the standard wisdom I've always heard is that _if you're talking about it you're losing_ and imposing lint in the build pipeline has a remarkable ability to prevent dialogue about style or format from emerging)

Reply:
I mean, would a pre-commit hook do? I can implement that